packages/tupaia-web/src/features/Map/Map.tsx

```diff
@@ -139,17 +131,10 @@ export const Map = () => {
     setActiveTileSet(TILE_SETS.find(({ key }) => key === tileSetKey) as typeof TILE_SETS[0]);
   };
 
-  const zoom = entity?.bounds ? undefined : 15;
-
   return (
     <MapContainer>
       <ErrorBoundary>
-        <StyledMap
-          center={entity?.point as LeafletMapProps['center']}
-          bounds={entity?.bounds as LeafletMapProps['bounds']}
-          zoom={zoom as LeafletMapProps['zoom']}
-          shouldSnapToPosition
-        >
+        <StyledMap shouldSnapToPosition>
```

Comment:
Just put a comment here that we are replacing the map-ui-components leaflet map positioning